src/plugins/saved_objects_management/public/management_section/objects_table/components/relationships.scss

```diff
@@ -0,0 +1,5 @@
+.savedObjectsManagementRelationships__title {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
We don't want to manually specify truncation styles - the current pattern in OpenSearch Dashboards appears to be 
```suggestion
  @include euiTextTruncate;
```

but I'd like to get more direct guidance from UX first, so I'm going to block this PR for now.

Reply:
@joshuarrrr We should not truncate headers, so I am not aligned with this change. OuiTitle should only wrap. Truncation applies to strings within OuiBreadcrumb, OuiBadge, OuiExpression, OuiBasicTable and OuiDataGrid.

Reply:
Also a +1 to not manually specifying truncation styles or usage. 

As the Design Systems team works towards recommendations for refinements in the available components, we are evaluating if any components should add the truncateText prop to avoid manual customization where we are seeing that its necessary. 
We have already determined that we do not want to add that prop to OuiTitle, or title-like components (OuiFlyoutHeader) or title props.

Reply:
Thanks for the information provided, will keep it in mind in the future. I removed the truncation and added the wrap text. You can see what it looks like now in the pictures below

<img width="700" alt="Screenshot 2023-06-05 at 15 17 54" src="https://github.com/opensearch-project/OpenSearch-Dashboards/assets/17855243/fe406672-eaa1-4102-8222-a0507915a0ec">
<img width="700" alt="Screenshot 2023-06-05 at 15 19 22" src="https://github.com/opensearch-project/OpenSearch-Dashboards/assets/17855243/46644be3-cf3d-4543-9501-d21c99c79ed2">